data/init.sql

```diff
@@ -0,0 +1,202 @@
+/*
+ * Set up the Initial Database for Us
+ * CON-In-A-Box 2018 Thomas Keeley
+ */
+
+-- SHOW DATABASES;
+-- SHOW TABLES;
+
+CREATE TABLE 'ConComList' (
+  'ListRecordID' INT UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  'AccountID' INT UNSIGNED NOT NULL,
+      -- Taken from NeonCRM Currently
+  'Department' INT UNSIGNED NOT NULL,
+  'Position' INT UNSIGNED NOT NULL,
+  'Note' VARCHAR(100),
+  'ConventionYear' INT UNSIGNED NOT NULL
+);
+
+CREATE TABLE 'ConComPositions' (
+  'PositionID' INT UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  'Name' VARCHAR(50) NOT NULL
+);
+
+INSERT INTO 'ConComPositions' ('Name') VALUES ('Head');
```

Comment:
So I am old fashion to DataBases and have not personally done one in many many years.
But I always liked having the structure and content of the DB be initialized separately. 

So have one sql file that creates the tables and then a second that inserts data into them.  This will also really help with the goal of making this expandable for future other cons.

Reply:
I can see that.  I usually use the init, or at least in this case, as the base line.  Basically seeding the data that could be changed in the future.  of course, it makes more sense as a script, sadly.  BUT, assuming we want a install and go solution, then there needs to be only one file to edit, update that with the DB info, and everything else comes from there.  Then you login to the website and configure the rest.  Was thinking kinda like wordpress, in a way.  

That said, having it in two files gives us options either way. I don't know if it would be helpful, but we can make it happen.  We could even update that init part to handle schema changes by doing "create table if not exist" followed by "alter table" to the "current" version.  Then NOT doing the base data import...  unless needed.  of course, the "right" way to do it would be to individually check if the tables exist, if not create.  Then check if that table's schema is right, if not, alter.  But I'm not ready for that in this version.  I want to focus on getting this to press since we are fast approaching our deadlines and you're waiting on me still. :)  SO... I can take that as a future enhancement but for now, focus on bas function.

Reply:
why did I have to think about the "right way".... sigh.  I implemented it.